mobile/src/libs/components/loader-container/loader-container.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
we should remove this component, but you can do this inside root navigation as it was before

Reply:
If I do as it was before, I won't be able to make the Loader transparent and show other pages underneath it, because the condition
```
   if (
    authenticatedUserDataStatus === DataStatus.IDLE ||
    authenticatedUserDataStatus === DataStatus.PENDING
  ) {
    return <Loader />;
  }
```
is triggered, and the Loader component is rendered, so there will be no way to see other pages. 
Also, if I leave it as it was before, I don't understand the point of the task because @k-mlch explained to me that the problem was precisely that the loader was on a white background, which looked strange between, for example, pages with a blue background. But if I changed the background to blue, then the loader would look strange between pages with a white background. That's why I decided to make it transparent

Reply:
you can put this condition into a variable and just return navigation like this 
```
<>
  <Navigation>
    // <Navigators />...
  <Navigation />
  <Loader isVisible={condition} />
</>
```

Reply:
Done